inc/Engine/CDN/CDN.php

```diff
@@ -108,14 +104,17 @@ public function rewrite_url( $url ) {
 			return rocket_add_url_protocol( $cdn_url . '/' . ltrim( $url, '/' ) );
 		}
 
-		$home       = get_option( 'home' );
+		$home       = home_url();
 		$home_parts = wp_parse_url( $home );
```

Comment:
The `$home` var isn't used past the next line of code. Right? As the function is named "home_url", it's clear what gets returned from it. Therefore, I think we can combine these 2 lines of code.

What are the advantages?

- Less memory
- Less processing
+ One less line of code to maintain

Reply:
https://github.com/wp-media/wp-rocket/pull/2589/commits/7c2ec6c8f0c776c7b86cf3a17d81daf192df446e